src/app/client/projects/desktop/src/app/modules/offline/components/offline-report-issues/offline-report-issues.component.spec.ts

```diff
@@ -1,25 +1,91 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
 import { OfflineReportIssuesComponent } from './offline-report-issues.component';
+import { ConfigService, ResourceService, BrowserCacheTtlService, ToasterService, SharedModule } from '@sunbird/shared';
+import { of as observableOf, throwError } from 'rxjs';
+import { SuiModalModule } from 'ng2-semantic-ui';
+import { CacheService } from 'ng2-cache-service';
+import { OfflineReportIssuesService } from '../../services/offline-report-issues/offline-report-issues.service';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
 
-xdescribe('OfflineReportIssuesComponent', () => {
+describe('OfflineReportIssuesComponent', () => {
   let component: OfflineReportIssuesComponent;
   let fixture: ComponentFixture<OfflineReportIssuesComponent>;
-
+  let resourceServiceStub;
   beforeEach(async(() => {
+    resourceServiceStub = {
+      instance: 'sunbird',
+      frmelmnts: {
+        lbl: {
+          issueReportedSuccessfuly: 'Issue reported successfully',
+          issueReportedSuccessfulySubNote: 'Note: Your report will be automatically sent to {instance} while online.'
+        }
+      }
+    };
     TestBed.configureTestingModule({
-      declarations: [ OfflineReportIssuesComponent ]
+      declarations: [OfflineReportIssuesComponent],
+      imports: [SuiModalModule, HttpClientTestingModule, SharedModule.forRoot()],
+      providers: [
+        ConfigService,
```

Comment:
dont add services of sharedmodule , all services will be added using sharedModule import